microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools-100.md

```diff
@@ -73,6 +73,35 @@ To learn more about the request log in the Network tool, see [Filter requests by
 For the history of this feature, see [Issue #16](https://github.com/MicrosoftEdge/DevTools/issues/16).
 
 
+<!-- ====================================================================== -->
+## Filter messages in the Issues tool by severity and browser target
+
+<!-- Title: Filter issues in the Issues tool -->
+<!-- Subtitle: New controls in the Issues tool allow you to filter messages by severity and browser target to help you focus on the most relevant feedback. -->
+
+In the **Issues** tool, the new **Severity** and **Browser** drop-down lists allow you to filter messages by severity and browser target, to help you focus on the most relevant feedback.  You can select or clear the options in each drop-down list.
+
+To see all relevant messages, you might need to refresh the page.  If prompted, in the **Issues** tool, click the **Regenerate issues** button.
+<!--
+The top-of-panel message after you change checkmarks on the drop down lists:
+"One or more settings have changed which require a panel reload to take effect.  [Regenerate issues]"
+-->
+
+By default, the **Severity** filter is set to show only warnings and errors:
+
+![Severity filter.](devtools-100-images/severity-filter.png)
+
+The **Browser** filter only applies to browser compatibility feedback, which is shown in the **Compatibility** section of the **Issues** list.  By default, the **Browser** filter shows messages for the top browsers, as defined by [Browserslist](https://github.com/browserslist/browserslist#queries):
+
+![The Browsers drop-down list, with "Top browsers" selected.](devtools-100-images/browser-filters-not-filtered.png)
+
+In the following example, messages about Firefox and Firefox for Android are filtered out:
+
+![Messages about Firefox browser filtered out (removed).](devtools-100-images/browser-filters-filtered-out.png)
+
+For more information, see [Filter issues](../../../issues/index.md#filter-issues) in _Find and fix problems using the Issues tool_.
```

Comment:
FYI, this anchor link did not scroll down to the relevant heading.

Reply:
It works on my machine; very high likelihood it's a temp caching issue involving review.dmc temp branch server.